CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 6.31.0 (February 25, 2023).
```

Comment:
25th is a Saturday 😄 Let's change to 27th and gives @danielmkn time to review as well.

Reply:
yea that was the goal I just got my dates way off XD